src/main.rs

```diff
@@ -18,14 +19,23 @@ use serenity::{
 pub mod channels;
 mod commands;
 const TOKEN: &str = "";
-use crate::commands::{PING_COMMAND, STUDY_COMMAND, UNSTUDY_COMMAND};
+use crate::commands::{PING_COMMAND, STUDY_COMMAND, UNSTUDY_COMMAND, MK_COMMAND};
```

Comment:
You can move the `group!` definition to the `comands.rs` file an import the entire group.